mne/io/eyelink/_utils.py

```diff
@@ -890,18 +908,37 @@ def _make_eyelink_annots(df_dict, create_annots, apply_offsets):
                 onsets = df["time"]
             durations = [0] * onsets
             descriptions = df["event_msg"]
+            this_annot = Annotations(
+                onset=onsets, duration=durations, description=descriptions
+            )
+        elif (key == "buttons") and (key in descs):
+            required_cols = {"time", "button_id", "button_pressed"}
+            if not required_cols.issubset(df.columns):
+                raise ValueError(f"Missing column: {required_cols - set(df.columns)}")
```

Comment:
This if condition is not covered by a test. I'm wondering, is there some  plausible scenario where we would expect  these column names (`'time'`, `'button_id'`, `'button_pressed'`) to be missing from the buttons dataframe? It looks like you hard code these column names during button dataframe creation, so I would expect that they will always be present? If so, I think this check is unnecessary.